solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttpSolrClient.java

```diff
@@ -259,6 +261,21 @@ public Builder withBaseSolrUrls(String... solrUrls) {
       return this;
     }
 
+    /**
+     * LBHttpSolrServer keeps pinging the dead servers at fixed interval to find if it is alive. Use
+     * this to set that interval
+     *
+     * @param interval time in milliseconds
+     */
+    public Builder setAliveCheckInterval(int interval) {
```

Comment:
use "with" builder naming convention

Reply:
I want to fix this as part of SOLR-16590 to get teh same name everywhere...